src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -21,22 +21,17 @@
 class SmsMessage implements MessageInterface, FromNotificationInterface
 {
     private ?string $transport = null;
-    private string $subject;
-    private string $phone;
-    private string $from;
-    private ?MessageOptionsInterface $options;
     private ?Notification $notification = null;
 
-    public function __construct(string $phone, string $subject, string $from = '', MessageOptionsInterface $options = null)
-    {
+    public function __construct(
+        private string $phone,
+        private string $subject,
+        private string $from = '',
```

Comment:
what about this one @wouterj , this should reverted AFAIU

Reply:
This one is correct. For both parameters in this method, the default value was on the parameter, not the property. This works exactly the same with and without CPP (we're only losing the property default value).

Reply:
Ah got it, thanks